test_quality_of_service/include/test_quality_of_service/qos_utilities.hpp

```diff
@@ -0,0 +1,138 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TEST_QUALITY_OF_SERVICE__QOS_UTILITIES_HPP_
+#define TEST_QUALITY_OF_SERVICE__QOS_UTILITIES_HPP_
+
```

Comment:
spurious empty line?

Reply:
Will fix, surprised the linters didn't complain.